requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy >= 1.16
+h5py >= 2.9
+matplotlib >= 3.0
```

Comment:
If this python package is going to be backward compatible with python 2.7, we need to relax this constraint. Matplotlib v3 is not available on python 2.7. Latest version for py27 is matplotlib 2.2.4